CondCore/CTPPSPlugins/test/testPPSTimingCalibration.sh

```diff
@@ -10,70 +10,54 @@ then
 
     echo "Testing history plots"
 
-    for db in 0 1
+    for db in 0 #1
```

Comment:
All these restricted ranges in the for loops were only intended for your private tests (and they should get removed from the version of the tests that has to be left in the release), or did you really intend to modify them centrally?

Reply:
It was modified on purpose - to test if the PI plugin was generated properly and is working it is enough to run test for only one combination (which is a lot faster); in case someone wants to run PI module locally to generate all possible plots it is possible to uncomment 'ranges' (since the code indicates what are all the posibilities), to generate all possible plots.

Reply:
I think limited one is enough - but ofc if you think the opposite I can change the test file.